tests/cypress/e2e/hedy_page/keyword_language.cy.js

```diff
@@ -0,0 +1,23 @@
+import {goToHedyPage} from "../tools/navigation/nav";
```

Comment:
Thanks for the added test, great to prevent this from happening in the future. This indeed fails on the current main so correctly catches the issue!!